docs/ossm/ossm2-migration/cluster-wide/README.md

```diff
@@ -47,45 +66,41 @@ Apart from the conditions above, it's recommended to decide which injection labe
 1. Prepare the `Istio` resource yaml named `ossm-3.yaml` to be deployed to the same namespace as the 2.6 control plane:
 
     Here we are not using any `discoverySelectors` so the control plane will have access to all namespaces. In case you want to define `discoverySelectors`, keep in mind that all data plane namespaces you are planning to migrate from 2.6 must be matched.
+    
+    Also note that `default` name with `InPlace` update strategy is used which allows usage of the `istio-injection=enabled` label. In case you want to use different name or `RevisionBased` update strategy, you will have to configure `default` `IstioRevisionTag`.
```

Comment:
Is naming the `Istio` "default" and using the `InPlace` strategy the only way to do this upgrade path?

Reply:
I think it should work also with the default tag.

Reply:
I'm not sure about usage of the `default` name because it is:

- a counterpart of the default installation of the upstream istio
- the simplest upgrade path which might be first option for many customers who are looking for simplicity

I agree we should somehow pushing customers to use more advanced configuration which will make the upgrade safer but I'm not sure if we should hide the simple options. Maybe we can add a note recommending usage of revision tags with revision based upgrade strategy?

Reply:
> I'm also not sure what happens when you create an InPlace with the name default and you create a default tag

You should never need the default revision tag in this case.